ccm_web/client/src/models/canvas.ts

```diff
@@ -6,5 +6,18 @@ export interface CanvasCourseBase {
 export interface CanvasCourseSection {
   id: number
   name: string
-  total_students?: number
+  total_students: number
 }
+
+export interface CanvasRole {
+  clientName: string
+  canvasName: string
+}
+
+export const canvasRoles: CanvasRole[] = [
```

Comment:
This contains the 'user friendly' role and the official canvas role names.  If we're going to be transforming user IDs in the backend now, maybe this only needs to be in the back end.

Reply:
Discussion with @chrisrrowland on slack, the FE would send the converted string to BE.